tools/mo/openvino/tools/mo/back/ResultRename.py

```diff
@@ -46,4 +46,7 @@ def find_and_replace_pattern(self, graph: Graph):
                     log.warning("Tensor name for Result node with name {} wasn't found. "
                                 "Default renaming was used: {}".format(node.soft_get('name', node.id),
                                                                        result_name))
+                for idx in range(len(graph.outputs_order)):
```

Comment:
Am I correct that due to this transformation `graph.outputs_order` gets useless since we lost original names of result nodes?

Reply:
Correct